packages/react-divider/src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { DividerProps } from './Divider.types';
+import { renderDivider } from './renderDivider';
+import { useDivider } from './useDivider';
+import { useDividerStyles } from './useDividerStyles';
+
+/**
+ * Define a styled Divider, using the `useDivider` and `useDividerStyles` hooks.
+ * {@docCategory Divider\}
```

Comment:
I still don't know why these backslashes were ever used in the first place but it is incorrect and results in docgen not actually working. Please remove

```suggestion
 * {@docCategory Divider}
```

Reply:
updated